app/src/main/java/com/concordia/mcga/utilities/pathfinding/GlobalPathFinder.java

```diff
@@ -84,9 +86,8 @@ private void outdoorToIndoorNavigation() throws MCGAPathFindingException {
     }
 
     private void externalOnlyNavigation() {
-        outdoorCoordinates = new LatLng[2];
-        outdoorCoordinates[0] = startPOI.getMapCoordinates();
-        outdoorCoordinates[1] = destPOI.getMapCoordinates();
+        OutdoorDirections outdoorDirections = activity.getNavigationFragment().getOutdoorDirections();
+        outdoorDirections.requestDirection(startPOI.getMapCoordinates(),destPOI.getMapCoordinates(),MCGATransportMode.TRANSIT);
```

Comment:
The navigation fragment should access the data available from GlobalPathFinder and call this, not the other way around